doc/python.md

```diff
@@ -140,7 +140,7 @@ would be fine.
 
 ### Packaging a job
 
-In order to be able to push a job to the the job server, it must be packaged into a Python Egg file. Similar to a Jar,
+In order to be able to push a job to the job server, it must be packaged into a Python Egg file. Similar to a Jar,
```

Comment:
should we mention about wheel file here?

Reply:
Good point, I missed that. I have adapted the paragraph.